hack/quick-install.sh

```diff
@@ -5,7 +5,16 @@
 set -o errexit
 set -o xtrace
 
-bash hack/install-database.sh
+kubectl apply -f - <<< $(helm template kubearchive charts/kubearchive \
+  -s templates/database/database.yaml \
+  --set "database.enabled=true"  --set "global.production=true")
+
+kubectl rollout status deployment --namespace=databases --timeout=30s
+sleep 15  # Wait for PostgreSQL to be really ready
```

Comment:
If you are going to have a pause like this, it is a good idea to put info on the screen so a user watching the progress knows why the script is pausing here.

```suggestion
echo "Waiting 15 seconds for the PostgreSQL database to finish initialization..."
sleep 15
```

Reply:
No sleep. Why not try to connect to the database in the script?

https://gist.github.com/nicerobot/1136dcfba6ce3da67ce3ded5101a4078 has some examples.

Reply:
I am thinking of adding a readiness probe in the deployment and remove the sleep, I think it will be easier to understand. For the first approach I used the sleep as it's what we are currently using in `hack/install-database.sh`, see https://github.com/kubearchive/kubearchive/blob/main/hack/install-database.sh#L99